pkg/katautils/katatrace/tracing.go

```diff
@@ -3,34 +3,47 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-package katautils
+package katatrace
 
 import (
 	"context"
 	"io"
 
 	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/sirupsen/logrus"
 	"github.com/uber/jaeger-client-go/config"
 )
 
 // Implements jaeger-client-go.Logger interface
 type traceLogger struct {
 }
 
+var kataTraceLogger = logrus.NewEntry(logrus.New())
+var tracingSet = false
+
 // tracerCloser contains a copy of the closer returned by createTracer() which
 // is used by stopTracing().
 var tracerCloser io.Closer
 
 func (t traceLogger) Error(msg string) {
-	kataUtilsLogger.Error(msg)
+	kataTraceLogger.Error(msg)
 }
 
 func (t traceLogger) Infof(msg string, args ...interface{}) {
-	kataUtilsLogger.Infof(msg, args...)
+	kataTraceLogger.Infof(msg, args...)
+}
+
+// SetLogger sets the logger to be used by katatrace, this fuction is
+// implicitly called by katautils.SetLogger
+func SetLogger(logger *logrus.Entry) {
+	kataTraceLogger = logger
 }
 
 // CreateTracer create a tracer
-func CreateTracer(name string) (opentracing.Tracer, error) {
+func CreateTracer(name string, tracing bool) (opentracing.Tracer, error) {
```

Comment:
See above comment; we can remove `tracing bool` if tracingSet is assigned in [pkg/katautils/config.go](https://github.com/kata-containers/runtime/blob/master/pkg/katautils/config.go#L1220-L1221)

Reply:
Fixed